packages/overlay/src/integrations/sentry/data/useSentryHelpers.tsx

```diff
@@ -1,13 +1,14 @@
 import { useContext } from 'react';
-import sentryDataCache from './sentryDataCache';
 import { SentryEventsContext } from './sentryEventsContext';
+import useSentryStore from './sentryStore';
```

Comment:
I think it might be worthwhile to provide a `getLocalTraces()` helper and refactor other places to show a "Local" badge to use something else. (this is for a follow up, not for this PR)